.circleci/config.yml

```diff
@@ -538,53 +592,56 @@ workflows:
           helm_repo: "https://charts.codacy.com/unstable"
           chart_version: $(cat .version)
           microk8s_channel: "1.16/stable"
-          helm_version: "v3.2.0"
+          helm_version: "v3.2.1"
+          helm_timeout: 900
+          context: CodacyDocker
+          docker_username: $DOCKER_USER
+          docker_password: $DOCKER_PASS
           <<: *helm_values
           requires:
             - helm_push_unstable
+
       - codacy/microk8s_install:
-          name: install_k8s-1.15_helm-3.2
+          name: install_k8s-1.17_helm-3.2
           helm_repo: "https://charts.codacy.com/unstable"
           chart_version: $(cat .version)
-          microk8s_channel: "1.15/stable"
-          helm_version: "v3.2.0"
-          <<: *helm_values
-          requires:
-            - helm_push_unstable
-      - codacy/microk8s_install:
-          name: install_k8s-1.14_helm-3.2
-          helm_repo: "https://charts.codacy.com/unstable"
-          chart_version: $(cat .version)
-          microk8s_channel: "1.14/stable"
-          helm_version: "v3.2.0"
+          microk8s_channel: "1.17/stable"
+          helm_version: "v3.2.1"
+          helm_timeout: 900
+          context: CodacyDocker
+          docker_username: $DOCKER_USER
+          docker_password: $DOCKER_PASS
           <<: *helm_values
           requires:
             - helm_push_unstable
+
       - codacy/microk8s_install:
-          name: install_k8s-1.13_helm-3.2
+          name: install_k8s-1.18_helm-3.2
           helm_repo: "https://charts.codacy.com/unstable"
           chart_version: $(cat .version)
-          microk8s_channel: "1.13/stable"
-          helm_version: "v3.2.0"
+          microk8s_channel: "1.18/stable"
+          helm_version: "v3.2.1"
+          helm_timeout: 900
+          context: CodacyDocker
+          docker_username: $DOCKER_USER
+          docker_password: $DOCKER_PASS
           <<: *helm_values
           requires:
             - helm_push_unstable
 
       - codacy/helm_promote:
-          name: promote_chart_to_stable
+          name: promote_chart_to_nightly
           context: CodacyHelm
           chart_name: codacy
           source_charts_repo_url: "https://charts.codacy.com/unstable"
           target_charts_repo_url: "https://charts.codacy.com/nightly"
           requires:
-            - install_k8s-1.13_helm-2.16
-            - install_k8s-1.14_helm-2.16
-            - install_k8s-1.15_helm-2.16
-            - install_k8s-1.16_helm-2.16
             - install_k8s-1.13_helm-3.2
             - install_k8s-1.14_helm-3.2
             - install_k8s-1.15_helm-3.2
             - install_k8s-1.16_helm-3.2
+            - install_k8s-1.17_helm-3.2
+            - install_k8s-1.18_helm-3.2
```

Comment:
This is really nice! The fact that we can automatically test that we can install on all these versions 👍